RecoHGCal/TICL/plugins/TrackstersMergeProducer.cc

```diff
@@ -672,6 +686,28 @@ void TrackstersMergeProducer::energyRegressionAndID(const std::vector<reco::Calo
   }
 }
 
+void TrackstersMergeProducer::assignTimeToCandidates(std::vector<TICLCandidate> &resultCandidates) const {
+  for (auto &cand : resultCandidates) {
+    if (cand.tracksters().size() > 1) {  // For single-trackster candidates the timing is already set
```

Comment:
Isn't this branch useless?
The next `for-loop` at line 694 will cover that `if` case automatically by skipping the loop completely.
The next ` if` at line 701 will protect the pathological cases and therefore cover also this branch-case.

Reply:
> Isn't this branch useless?
> The next `for-loop` at line 694 will cover that `if` case automatically by skipping the loop completely.

Not in case the `cand.tracksters().size()` is exactly equal to `1`...

> The next ` if` at line 701 will protect the pathological cases and therefore cover also this branch-case.